analysisCode/src/EventLoop.cpp

```diff
@@ -1,51 +1,59 @@
-/// Fastjet includes
 #include <fastjet/ClusterSequence.hh>
 #include <fastjet/Selector.hh>
 #include <fastjet/contrib/SoftDrop.hh>
+#include <iostream>
 
-/// Root includes
 #include <TFile.h>
 #include <TTree.h>
 #include <TSystem.h>
-
-/// EIC smear includes
 #include <eicsmear/erhic/EventPythia.h>
 #include <eicsmear/smear/EventS.h>
-
-/// Local includes
 #include "include/TruthEvent.h"
 #include "include/SmearedEvent.h"
 #include "include/JetDef.h"
 #include "include/SoftDropJetDef.h"
 
-#include <iostream>
+using namespace std;
+using namespace fastjet;
 
-int main(int argc, char **argv)
+int main()
```

Comment:
Let's leave the arguments as they were. We will likely be running this executable on condor and thus will want the ability to run with arguments that correspond to e.g. different files.

Reply:
Agreed, I switched it for simplicity for testing purposes (just because I had not figured out how to use the executable function with the new arguments). It was a mistake that I did not change this back. In practice, I would not have wanted to make a pull request for this file, but am still getting used to git. I realize I could have just not committed it before making the pull request, I will keep this in mind for the future.